modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/application/DefaultMuleApplicationStatusTestCase.java

```diff
@@ -72,8 +73,8 @@ protected void doSetUp() throws Exception {
     when(mockArtifactContext.getMuleContext()).thenReturn(muleContext);
     when(mockArtifactContext.getRegistry()).thenReturn(new DefaultRegistry(muleContext));
     ApplicationDescriptor applicationDescriptorMock = mock(ApplicationDescriptor.class, RETURNS_DEEP_STUBS.get());
-    when(applicationDescriptorMock.getClassLoaderModel())
```

Comment:
allure at test level

Reply:
What kind of allure? It's not actually testing anything about the `ClassLoaderConfiguration`.